app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -651,7 +629,8 @@ class TopicFragmentTest {
     )
   }
 
-  private fun initializeApplicationComponent() {
+  private fun initializeApplicationComponent(enableExtraTabsUi: Boolean = false) {
```

Comment:
Suggest not defaulting (since you aren't relying on it, anyway, and it forces tests to pick their behaviors which is generally better, anyway).

Reply:
done

Reply:
Doesn't seem resolved--the parameter still has a default.

Reply:
Resolved now.